drivers/soundwire/amd_init.c

```diff
@@ -15,6 +15,47 @@
 
 #include "amd_init.h"
 
+#define ACP_PAD_PULLDOWN_CTRL				0x0001448
+#define ACP_SW_PAD_KEEPER_EN				0x0001454
+#define AMD_SDW_PAD_PULLDOWN_CTRL_ENABLE_MASK		0x7f9a
+#define AMD_SDW_PAD_KEEPER_EN_MASK			0x11
```

Comment:
AMD_SDW0_PAD_KEEPER_EN_MASK |  AMD_SDW1_PAD_KEEPER_EN_MASK

Reply:
will fix it.